qp_qiime2/tests/test_qiime2.py

```diff
@@ -102,12 +103,20 @@ def test_rarefy(self):
         success, ainfo, msg = call_qiime2(self.qclient, jid, params, out_dir)
         self.assertTrue(success)
         self.assertEqual(msg, '')
-        self.assertEqual(ainfo[0].files, [(
-            join(out_dir, 'rarefy', 'rarefied_table', 'feature-table.biom'),
-            'biom')])
+        obs_fp = join(out_dir, 'rarefy', 'rarefied_table',
```

Comment:
'rarefy' might be a misspelling of 'rarify'.

Reply:
nop and note that those names come directly from qiita